client/src/pages/dashboard/Dashboard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+import Navbar from '../layout/Navbar';
+
+const useStyles = makeStyles(theme => ({
+    sidebar: {
+        height: '85vh',
```

Comment:
Is this to keep the sidebar from going under the navbar?  If so, you should be able to structure the layout to avoid the need for this.

Reply:
No, this is a temporary height to see 'flexGrow' properties of the inside divs, since I haven't put the actual content in there yet the grid shrinks to the height of the sample texts.